.forge-snapshots/simple swap.snap

```diff
@@ -1 +1 @@
-124313
\ No newline at end of file
+124651
```

Comment:
!!!!!!! 350!! this makes me want 2 separate functions more 👀

Reply:
do you want to overload `settle()` and `settle(address)`?